openlcsd/flow/tasks.py

```diff
@@ -931,7 +931,7 @@ def save_group_components(context, engine):
     msg = f'Start to save components in {component_type} {package_nvr}...'
     engine.logger.info(msg)
     fd, tmp_file_path = tempfile.mkstemp(
-        prefix='scan_group_components_', dir=context.get('post_dir'))
+        prefix='save_components_', dir=context.get('post_dir'))
```

Comment:
Whether to modify the name of the function `save_group_components` to `save_components` ?

Reply:
OK